dashboard/pubspec.yaml

```diff
@@ -19,8 +19,7 @@ dependencies:
   fixnum: 1.1.0 # Rolled by dependabot
   flutter_app_icons: 0.0.9 # Rolled by dependabot
   google_sign_in: 6.2.1 # Rolled by dependabot
-  google_sign_in_platform_interface: any # Match google_sign_in
-  google_sign_in_web: 0.12.3 # Rolled by dependabot
+  google_sign_in_web: 0.12.3+2 # Rolled by dependabot
```

Comment:
is it possible dependabot reverts this?

Reply:
I'll give it a try with a dependabot manual run to see if it will try to revert the change

Reply:
I'm trying to see if we want to move the `_web` package to something `> 1.0.0` so this isn't an issue anymore.

However, if we don't update it to `1.0.0`, and then don't do any backwards-incompatible changes, it's likely that this can stay as `0.12.Y+Z` for a while!